pom.xml

```diff
@@ -32,6 +46,44 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>net.adoptopenjdk.api</groupId>
+      <artifactId>adoptopenjdk-api-v3-models</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-reflect</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-kotlin</artifactId>
+      <version>2.12.0</version>
+    </dependency>
+    <dependency>
+      <groupId>net.adoptopenjdk</groupId>
+      <artifactId>net.adoptopenjdk.v3.api</artifactId>
+      <version>0.4.0</version>
+    </dependency>
+    <dependency>
+      <groupId>net.adoptopenjdk.api</groupId>
+      <artifactId>adoptopenjdk-api-v3-models</artifactId>
```

Comment:
This PR adds 2 dependencies to `adoptopenjdk-api-v3-models`. I assume one can be removed?